toolchain/oprofile.py

```diff
@@ -35,9 +35,11 @@ def setUp(self):
             deps.append("git-core")
         else:
             deps.append("git")
-        if detected_distro.name in ["Ubuntu", 'debian']:
-            deps.append("libxml2-utils")
-            deps.append("tclsh")
+        if detected_distro.name in ["Ubuntu"]:
+            deps.append(["libxml2-utils", "tclsh", "oprofile"])
```

Comment:
here also you can extend package name for Ubuntu.
```
if detected_distro.name in ["Ubuntu", "debian"]:
    deps.extend(["libxml2-utils", "tclsh"])
    if detected_distro.name == "Ubuntu":
        deps.extend(["oprofile"])
```

And also note than the `append` is wrong here. It has to be `extend` if you are using a list of packages.